main.py

```diff
@@ -141,8 +142,12 @@ def load_user(userid):
     login_manager.anonymous_user = load_anonymous_user
 
     stripe.api_key = app.config["STRIPE_SECRET_KEY"]
-    pywisetransfer.environment = app.config["TRANSFERWISE_ENVIRONMENT"]
-    pywisetransfer.api_key = app.config["TRANSFERWISE_API_TOKEN"]
+    global wise
+    wise = pywisetransfer.Client(
+        api_key=app.config["TRANSFERWISE_API_TOKEN"],
+        environment=app.config["TRANSFERWISE_ENVIRONMENT"],
+        private_key_file=app.config.get("TRANSFERWISE_PRIVATE_KEY_FILE"),
```

Comment:
Checking my understanding: this is a `get` instead of a direct index lookup so that the app can be configured to work in test/sandbox environments without a private key for SCA available?

Reply:
Yep, and the API_TOKEN is defaulted to an empty string in the config example, so I assume it's not intended to be optional.